napari/layers/_layer_actions.py

```diff
@@ -132,7 +132,22 @@ def _convert_dtype(ll: LayerList, mode='int64'):
         layer.data = layer.data.astype(np.dtype(mode))
 
 
-def _project(ll: LayerList, axis: int = 0, mode='max'):
+def _project(ll: LayerList, dims: Dims, axis: int = 0, mode='max'):
+    """Creates a new layer with specified projection.
```

Comment:
Thanks for adding the docstring!

I think this option will only be available for 3D images at the moment, so it might be worth mentioning that in the docstring.

Reply:
True, will add.